src/promptflow-azure/promptflow/azure/_utils/general.py

```diff
@@ -2,11 +2,23 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
+import typing
+
 import jwt
+from azure.ai.ml import MLClient
+from azure.ai.ml.entities import Workspace
+from azure.identity import AzureCliCredential, DefaultAzureCredential
 
+from promptflow._sdk._utils import extract_workspace_triad_from_trace_provider
 from promptflow.core._connection_provider._utils import get_arm_token, get_token
 
 
+class WorkspaceKind:
+    DEFAULT = "default"
+    HUB = "hub"
+    PROJECT = "project"
```

Comment:
Can also apply these constants for `_workspace_default_datastore`

Reply:
applied